pkg/storage/runtime_test.go

```diff
@@ -32,14 +32,16 @@ var _ = t.Describe("Runtime", func() {
 		gomock.InOrder(
 			imageServerMock.EXPECT().GetStore().Return(storeMock),
 			storeMock.EXPECT().Image(gomock.Any()).Return(&cs.Image{}, nil),
```

Comment:
AFAICS this corresponds to https://github.com/openSUSE/cri-o/blob/e758da45bd7365196e7148425a6405830ebbdfc9/vendor/github.com/containers/image/storage/storage_transport.go#L151 , which tries to interpret the name as an image ID, and that should fail with `(nil, cstorage.ErrImageUnknown)`.